scripts/deploy/package-webapi.sh

```diff
@@ -100,6 +107,41 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+if [[ -z "$NO_FRONTEND" ]]; then
+    echo "Building static frontend files..."
+
+    ENV_FILE_PATH="$SCRIPT_ROOT/../../webapp/.env"
+    echo "Writing environment variables to '$ENV_FILE_PATH'..."
+    echo "REACT_APP_BACKEND_URI=<-=TOKEN=->Frontend:BackendUri</-=TOKEN=->" > $ENV_FILE_PATH
+    echo "REACT_APP_AUTH_TYPE=AzureAd" >> $ENV_FILE_PATH
+    echo "REACT_APP_AAD_AUTHORITY=<-=TOKEN=->Authentication:AzureAd:Instance</-=TOKEN=->/<-=TOKEN=->Authentication:AzureAd:TenantId</-=TOKEN=->" >> $ENV_FILE_PATH
+    echo "REACT_APP_AAD_CLIENT_ID=<-=TOKEN=->Frontend:AadClientId</-=TOKEN=->" >> $ENV_FILE_PATH
+    echo "REACT_APP_AAD_API_SCOPE=api://<-=TOKEN=->Authentication:AzureAd:ClientId</-=TOKEN=->/<-=TOKEN=->Authentication:AzureAd:Scopes</-=TOKEN=->" >> $ENV_FILE_PATH
```

Comment:
as you know, we can remove this now!

Reply:
Yep.  Done on a local copy that I'm testing right now.